source/routing/rendering-a-template.md

```diff
@@ -25,16 +23,7 @@ export default Ember.Route.extend({
 });
 ```
 
-If you want to use a different controller than the route handler's
-controller, pass the controller's name in the `controller` option:
-
-```app/routes/post.js
-export default Ember.Route.extend({
-  renderTemplate() {
-    this.render({ controller: 'favoritePost' });
-  }
-});
-```
+## Named Outlets
```

Comment:
We should confirm if components replace all use cases of named outlets and if this section should be removed from the guide. cc @rwjblue